src/common/actions/repositories.js

```diff
@@ -11,10 +11,13 @@ export const REPOSITORIES_SUCCESS = 'REPOSITORIES_SUCCESS';
 export const REPOSITORIES_FAILURE = 'REPOSITORIES_FAILURE';
 
 
-export function fetchRepositoriesSuccess(repos) {
+export function fetchRepositoriesSuccess(result) {
   return {
     type: REPOSITORIES_SUCCESS,
-    payload: repos
+    payload: {
+      ...result.payload,
+      pageLinks: result.pageLinks
```

Comment:
Following conventions from #464 actions should know as little as possible about structure of the response and just pass response payload to reducer if possible.

Maybe refactoring it to use callApi middleware would help with that.